build/vs_toolchain.py

```diff
@@ -36,8 +36,8 @@
 
 # VS versions are listed in descending order of priority (highest first).
 MSVS_VERSIONS = collections.OrderedDict([
-  ('2017', '15.0'),
   ('2019', '16.0'),
+  ('2017', '15.0'),
```

Comment:
Why was this line (2017) swapped with (2019)

Reply:
Yup that's right, these are ordered by priority. There's a comment above this, let me know if you have suggestions to improve clarity:

https://github.com/flutter/buildroot/blob/master/build/vs_toolchain.py#L37